.github/workflows/pages-prod.yml

```
@@ -0,0 +1,27 @@
+name: Deploy to Production
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)